components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.admin.apis/AdminApiCapp/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is there a particular reason to mention '2018' as the year instead of '2019'?

Reply:
Yes. This code was moved from product-ei which was written in 2018. Therefore we need to keep the original license header.